src/main/java/app/servlets/ControllerServlet.java

```diff
@@ -0,0 +1,232 @@
+package app.servlets;
+
+import app.dbService.DBException;
+import app.dbService.PostgresBookService;
+import app.dbService.entity.Book;
+import org.apache.tomcat.util.http.fileupload.FileItem;
+import org.apache.tomcat.util.http.fileupload.FileItemFactory;
+import org.apache.tomcat.util.http.fileupload.FileUploadException;
+import org.apache.tomcat.util.http.fileupload.disk.DiskFileItemFactory;
+import org.apache.tomcat.util.http.fileupload.servlet.ServletFileUpload;
+import org.apache.tomcat.util.http.fileupload.servlet.ServletRequestContext;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.*;
+
+import org.apache.commons.codec.binary.Base64;
+
+import java.sql.SQLException;
+import java.util.Iterator;
+import java.util.List;
+
+public class ControllerServlet extends HttpServlet {
+    private final PostgresBookService service = new PostgresBookService();
+    private Book book;
```

Comment:
pls check comments for previous commits, since you moved current class, there are lots of them left behind, so you might have missed them